lib/cli/generators/METEOR/index.js

```diff
@@ -44,14 +44,14 @@ export default async npmOptions => {
     babelrc = JSON5.parse(babelrcContent);
     babelrc.plugins = babelrc.plugins || [];
 
-    if (babelrc.plugins.indexOf('module:babel-root-slash-import') < 0) {
```

Comment:
I had a warning from Babel 7 that told me to add this `module:`. Is there a chance that now it runs with Babel 6 instead?

Reply:
For some reason, there's no `babel-loader` here: https://github.com/storybooks/storybook/blob/f8332e744b56fd4182eeba86538223f8bf43f139/lib/cli/test/snapshots/meteor/package.json

Reply:
Looks like I just forgot to change METEOR on https://github.com/storybooks/storybook/commit/3c0e3832e169e5e874c1bc774b7b68e652c1c803

Reply:
> But this did not come up in CircleCI

This is because you changed only circleCI config when introducing e2e

Reply:
Seems to be a completely unnecessary plugin that's 3 years old, and unmaintained..
If everything is green with this gone, I'm just removing it.

Reply:
Please also add `installBabelLoader` in meteor